etc/firebase-admin.api.md

```diff
@@ -498,15 +498,18 @@ export interface GoogleOAuthAccessToken {
 export function initializeApp(options?: AppOptions, name?: string): app.App;
 
 // @public
-export function instanceId(app?: app.App): instanceId.InstanceId;
+export class InstanceId {
+    get app(): App;
+    deleteInstanceId(instanceId: string): Promise<void>;
+    }

```

Comment:
This is generated from the api-extractor, isn't it? Is there an indentation issue here?

Reply:
Yes, this is auto-generated. Not sure what we can do about the indentation issue. Everything looks good in source and the generated d.ts files.